EIPS/eip-7775.md

```diff
@@ -0,0 +1,100 @@
+---
+eip: 7775
+title: New opcode BURN
+description: An opcode to burn native token at the given address
+author: Dev Bear (@itsdevbear)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-09-30
+---
+
+## Abstract
+
+This proposal introduces a new `BURN` opcode, designed to burn native tokens at the given address.
+
+## Motivation
+
+The motivation for this proposal is to provide a standardized and efficient way to burn native tokens directly within the EVM. Historically, contracts such as the BeaconDepositContract have "burned" ether by making it irrecoverable from the given address. This approach can lead to confusion and potential misuse. By introducing a dedicated `BURN` opcode, we can ensure a clear and consistent method for burning native tokens. This could become useful for Ethereum L2s when transferring ether back to the L1, as well as other EVM L1 chains that could leverage this for their cryptoeconomics. Additionally, it opens up an interesting design space for [EIP-7685](./eip-7685.md) use cases, providing a more robust and standardized mechanism for token management.
+
+## Specification
+
+The `BURN` opcode `(0xF8)` is introduced with the following behavior:
+
+1. Pops one 32-byte word from the top of the stack, treating this value as the `uint256` value of native token to be burned.
+2. Retrieves the current address from the EVM execution context.
+3. Checks the balance of the current address.
+4. If the value to be burned is greater than the balance of the current address, the opcode `MUST` revert.
+5. If the value to be burned is 0, the execution `MUST NOT` revert.
+6. Subtracts the value from the current address's native token balance.
+
+When `BURN` is used in the context of a DELEGATECALL or CALLCODE, the contract whose balance is to be manipulated is the contract that issued the DELEGATECALL or CALLCODE instruction.
```

Comment:
I think a note here that it is not possible to use `BURN` in a context where `STATICCALL` is used should be added.